lib/plugin/utils/plugin-utils.ts

```diff
@@ -45,7 +46,7 @@ export function getTypeReferenceAsString(
   if (isNumber(type)) {
     return Number.name;
   }
-  if (isString(type)) {
+  if (isString(type) || isStringLiteral(type)) {
```

Comment:
This is the actual fix, the rest is formatting changes.

Reply:
If the rest is formatting changes, can you please revert them and only leave the actual fix + added tests?

Reply:
The formatting changes was the result of running `npm run format` as detailed in the `CONTRIBUTING.md` doc. I can revert, but it seems like this is the right thing to do before opening a PR?